extmod/moducryptolib.c

```diff
@@ -44,10 +44,21 @@ enum {
     UCRYPTOLIB_MODE_MIN = 0,
     UCRYPTOLIB_MODE_ECB,
     UCRYPTOLIB_MODE_CBC,
+#if MICROPY_PY_UCRYPTOLIB_CTR
+    UCRYPTOLIB_MODE_CTR = 6,
+#endif
     UCRYPTOLIB_MODE_MAX,
 };
 
+struct ctr_params {
+    // nonce is the IV of the AES context.
+
```

Comment:
What does this empty line mean?

Reply:
If adjacent to the next line (`nc_off`), I read it as referring to that line. It's not, so I've added a blank.